doc/classes/Theme.xml

```diff
@@ -277,14 +313,17 @@
 			</argument>
 			<argument index="1" name="type" type="String">
 			</argument>
-			<argument index="2" name="texture" type="StyleBox">
+			<argument index="2" name="stylebox" type="StyleBox">
```

Comment:
You can't edit argument tags - these are automatically generated from source. Only `<description>` tags may be edited.